### ISSUE TITLE
Reword scheduled delivery frequency descriptions to be a bit more compact

### ISSUE THREAD

**Issue by PriPatel** [lightdash/lightdash#5183]
<img width="128" alt="Screenshot 2023-04-21 at 15 52 49" src="https://user-images.githubusercontent.com/12776522/233628356-32592462-b755-4c68-bdfe-0952c8a99fe5.png">
The frequency descriptions are a bit wordy and feel unnatural

eg the second one could read `Mondays at 9am`.

**Comment by PriPatel** [comment]
Thanks @ZeRego that could be a good option. Is there an easy way to see what the smaller descriptions might look like? Or do you reckon we just turn off `verbose` and check it out?

**Comment by ZeRego** [comment]
@PriPatel do you have specific examples you want to compare ? if so, let me know and I can quickly provide you the verbose and non verbose output.

**Comment by PriPatel** [comment]
@ZeRego thanks! Here are a few that could be less verbose: 

`at 12:15 AM, on day 1 of the month`
`at 05:00 PM, every day
``at 08:57 AM, only on Monday`

**Comment by ZeRego** [comment]
The only difference was `At 05:00 PM, every day` and `At 05:00 PM`, the rest was the same.

Unit tests if we want to test more expressions
```
test('cron expressions', async () => {
    expect(
        cronstrue.toString('15 0 1 * *', {
            verbose: true,
            throwExceptionOnParseError: false,
        }),
    ).toEqual('At 12:15 AM, on day 1 of the month');
    expect(
        cronstrue.toString('15 0 1 * *', {
            verbose: false,
            throwExceptionOnParseError: false,
        }),
    ).toEqual('At 12:15 AM, on day 1 of the month');

    expect(
        cronstrue.toString('0 17 * * *', {
            verbose: true,
            throwExceptionOnParseError: false,
        }),
    ).toEqual('At 05:00 PM, every day');
    expect(
        cronstrue.toString('0 17 * * *', {
            verbose: false,
            throwExceptionOnParseError: false,
        }),
    ).toEqual('At 05:00 PM');

    expect(
        cronstrue.toString('57 8 * * 1', {
            verbose: true,
            throwExceptionOnParseError: false,
        }),
    ).toEqual('At 08:57 AM, only on Monday');
    expect(
        cronstrue.toString('57 8 * * 1', {
            verbose: false,
            throwExceptionOnParseError: false,
        }),
    ).toEqual('At 08:57 AM, only on Monday');
});
```
